flow/record/base.py

```diff
@@ -96,6 +96,18 @@ def _unpack(__cls, {args}):
 """
 
 
+if env_excluded_fields := os.environ.get("FLOW_RECORD_IGNORE"):
+    IGNORE_FIELDS_FOR_COMPARISON = env_excluded_fields.split(",")
+else:
+    IGNORE_FIELDS_FOR_COMPARISON = []
+
+
+def update_ignored_fields_for_comparison(ignored_fields: list[str]) -> None:
+    """Can be used to update the IGNORE_FIELDS_FOR_COMPARISON from outside the flow.record package scope"""
+    global IGNORE_FIELDS_FOR_COMPARISON
+    IGNORE_FIELDS_FOR_COMPARISON = ignored_fields
```

Comment:
I think `set_ignored_fields_for_comparison()` is a bit more clearer than `update_ignored_fields_for_comparison()`. The latter would make me think that it does something with the previous value.

Also using `set()` for efficiency and filter out possible duplicate field names.

```suggestion
def set_ignored_fields_for_comparison(ignored_fields: list[str]) -> None:
    """Can be used to update the IGNORE_FIELDS_FOR_COMPARISON from outside the flow.record package scope"""
    global IGNORE_FIELDS_FOR_COMPARISON
    IGNORE_FIELDS_FOR_COMPARISON = set(ignored_fields)
```